src/classes/CW_AssociateAudienceRewardHelperTest.cls

```diff
@@ -0,0 +1,124 @@
+@isTest
+private class CW_AssociateAudienceRewardHelperTest 
+{
+
+	static testMethod void myUnitTest() 
```

Comment:
Style issue: test methods should be annotated with @IsTest for better readibility